urlextract/urlextract_core.py

```diff
@@ -66,18 +66,18 @@ class URLExtract(CacheFile):
         ("`", "`"),
     }
 
-    _ipv4_tld = ['.{}'.format(ip) for ip in reversed(range(256))]
+    _ipv4_tld = [".{}".format(ip) for ip in reversed(range(256))]
     _ignore_list = set()
 
     _limit = DEFAULT_LIMIT
 
     def __init__(
-            self,
-            extract_email=False,
-            cache_dns=True,
-            extract_localhost=True,
-            limit=DEFAULT_LIMIT,
-            **kwargs
+        self,
+        extract_email=False,
+        cache_dns=True,
+        extract_localhost=True,
+        limit=DEFAULT_LIMIT,
+        **kwargs
```

Comment:
@za is the comma added here automatically by black? Or what happen?

Reply:
@lipoja Yes, Black (suggested) to have comma. Previously it wasn't. It all started with this commit: https://github.com/lipoja/URLExtract/pull/105/commits/866ff39db2e07b922c4e8c075b16c171e6ab1e69 (see the commit message for detail).